### ISSUE TITLE
Need explanation of inputs

### ISSUE THREAD

**Issue by ssgari** [youkely/InfrasCal#20]
Can anyone please explain what the inputs in /data are needed? Am I correct that all you need is timestamp-aligned images from each camera in your setup (given the cameras have overlapping FOVs)? Do we need some kind of point cloud as input?
Also, what does the "vocabulary" input refer to?

**Comment by youkely** [comment]
Yes, you need timestamp-aligned images for each camera(not necessarily having overlapping FOVs). The map is stored in data/demo/map which is in colmap format. Vocabulary is for feature matching.
